packages/email-nodemailer/src/index.ts

```diff
@@ -66,7 +66,7 @@ async function buildEmail(emailConfig?: NodemailerAdapterArgs): Promise<{
     transport = await createMockAccount(emailConfig)
   }
 
-  if (emailConfig.skipVerify !== false) {
+  if (emailConfig.skipVerify === false) {
```

Comment:
This also needs to handle when `skipVerify` is undefined. It should verify when undefined.